docs/data-structures/postordertraversal.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Postorder traversal of Binary Tree
+slug: /data-structures/postordertraversal
+---
+
+## Introduction
+
+Binary Tree is a type of tree which have at most two children or we can say at most degree 2. So to traverse the binary tree according to Depth-first search. We have a traversal algorithm called Postorder traversal.
+
+## Explanation 
+In this type of traversal(Postorder traversal). first, we have to visit left part then right part then its root respectively. In this first visit left part that means we have to traverse the whole left subtree in this order (left->right->root) after that whole right subtree in this order (left->right->root) and then visit root of that subtree. Postorder: (left->right->root). 
+#### Example 1
+
+<img src="https://i.imgur.com/f0nHPpz.jpg?1" width="600" height="350"/>
+
+```
+
+ First traverse the left subtree: Node(44).
+ Then its left subtree: Node(22)
+ Again it has left child
+ Visit its left: Node(11)
+ Postorder traversal: 11
+ Visit right child of node(22): Node(33)
+ Postorder traversal: 11 33
+ Visit its root: Node(22)
+ Postorder traversal: 11 33 22 
+ Right of this subtree parent root(Node(44)): Node(66)
+ Left subtree of this root(Node(66)): Node(55)
+ Right child of this subtree: Node(77)
+ Postorder traversal: 11 33 22 55 77
+ Root of this subtree: Node(66)
+ Postorder traversal: 11 33 22 55 77 66
+ Root of the right subtree: Node(44)
+ Now our Postorder traversal: 11 33 22 55 77 66 44 !!
+
+```
+## Algorithm
+
+For Postorder traversal, we follow the following algorithm:
+
+* First, we have to traverse the Binary tree in this order(left->right->root).
+
+* Left means traversing the whole left subtree in this order(left->right->root) until we get left child as null. 
+
+* Root means parent of that particular subtree.
+
+* Right means traverse the whole right subtree in this (left->right->root) order until we get null.
```

Comment:
Make it: For Postorder traversal, we follow the following algorithm:

First, we have to traverse the Binary tree in this order(left->right->root).

Left means traversing the whole left subtree in this order(left->right->root) until we get the left child as null.

Root means the parent of that particular subtree.

Right means traverse the whole right subtree in this (left->right->root) order until we get null.